syntax = "proto3";

package verify;    // 类似于一个 namespace 命名空间

service VerifyService{  // gRPC的服务接口
    // 方法 RPC 方法 GetVerifyCode，接受 GetVerifyReq 类型的请求，并返回 GetVerifyRsp 类型的响应。
    rpc GetVerifyCode (GetVerifyReq) returns (GetVerifyRsp) {}
}

message GetVerifyReq{
    string email = 1;
}

message GetVerifyRsp{
    int32 error = 1;
    string email = 2;
    string code = 3;
}